app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -1681,6 +1681,9 @@ class BrowserTabFragment :
                 duckPlayerScripts.sendSubscriptionEvent(it.duckPlayerData)
             }
             is Command.SwitchToTab -> {
+                binding.focusedView.gone()
```

Comment:
Curious why we need this, since it doesn't seem to be related to the migration

Reply:
This code was added to cover the scenario where the `Switch to tab` suggestion matches the current tab. In this case we need to close the suggestions and have the UI in a state similar as when opening a different tab.